index.php

```diff
@@ -9,13 +9,27 @@
 $PDO = connectDb($host, $username, $password, $dbName);
 
 $errorMessage = "";
-if(count($_POST)>0) {
+if(count($_POST) == 4) {
     $formSubmissionCheck = checkFormSubmission($_POST);
     $dropdownSubmissionCheck = checkDropdownSubmission($_POST);
     if ($formSubmissionCheck['result'] && $dropdownSubmissionCheck['result']) {
         addBottle($PDO, $_POST['item-name'], $_POST['purchase-location'], $_POST['type'], $_POST['purchase-date']);
     } else $errorMessage = $formSubmissionCheck['message'] . $dropdownSubmissionCheck['message'];
 }
+if(count($_POST) == 5) {
```

Comment:
better to have a special POST field that indicates whether it's a new item or an edit, rather than inferring from the number of fields

Reply:
but not required for this project